modules/core/src/main/scala/com.snowplowanalytics.snowplow.lakes/processing/Processing.scala

```diff
@@ -129,6 +130,20 @@ object Processing {
       }
     }.drain
 
+  private def setLatency[F[_]: Sync](metrics: Metrics[F]): Pipe[F, TokenedEvents, TokenedEvents] =
+    _.evalTap {
```

Comment:
Do we use a `Pipe` and `evalTap` instead of `observe` because the docs say that this is faster or is there another reason?